src/jsx/app/app.jsx

```diff
@@ -1,3 +1,7 @@
+var Router = window.ReactRouter;
```

Comment:
Yup. All things client is installed in the global, unless it's a jQuery plugin. Then it would be appended to the $.